tools/extra/qpafilter/qpafilter.py

```diff
@@ -152,57 +240,70 @@ def read_qpa(in_file, temp_overrides):
        'fatal', 'warning', 'units'}.
     """
     override_d = {}
-    for ot in temp_overrides:
+    for override in temp_overrides:
         try:
-            olabel, opercentage = ot.split(':')
+            olabel, opercentage = override.split(':')
         except ValueError:
-            LOG.warning(f'{ot} is not a valid temperature '
+            LOG.warning(f'{override} is not a valid temperature '
                         f'override. Skipping')
-            continue
-        override_d[olabel] = opercentage
+            continue         # percentage, was it applied?
+        override_d[olabel] = [opercentage, False]
 
-    category_re = re.compile(CATEGORY_PATTERN, re.DOTALL|re.UNICODE)
+    category_re = re.compile(CATEGORY_PATTERN, re.DOTALL | re.UNICODE)
     item_re = re.compile(DATA_ITEM_PATTERN, re.UNICODE)
     outer_d = defaultdict(list)
-    for mc in category_re.finditer(in_file.read()):
-        category = mc.group('category').strip()
-        for mv in item_re.finditer(mc.group('values')):
-            label = mv.group('label').strip()
-            fatal = mv.group('value').strip()
-            units = mv.group('units').strip()
+    for cat_match in category_re.finditer(in_file.read()):
+        category = cat_match.group('category').strip()
+        for val_match in item_re.finditer(cat_match.group('values')):
+            label = val_match.group('label').strip()
+            fatal = val_match.group('value').strip()
+            units = val_match.group('units').strip()
 
             inner_d = {'label': label,
                        'fatal': fatal,
                        'units': units,
-                       'warning': 0.0 # placeholder
-                      }
+                       'warning': 0.0}  # placeholder
 
             if label in override_d:
-                inner_d['override'] = override_d[label]
-                LOG.info(f'Setting override for \'{label}\' to {inner_d["override"]}%')
+                override_d[label][1] = True
+                inner_d['override'] = override_d[label][0]
```

Comment:
Another way you can track whether an override has been used is to pop it out of the container object (dict).
```Python
            if label in override_d:
                inner_d['override'] = override_d.pop(label)
```